wordpress-mobile/WordPress-Android
# issue-12442/ added events for bottom sheet.

Fixes #12315

## Solution
Added analytics events to track when the bottom sheet is opened and when the publish button is clicked. 

## Testing
1. Create a new story post. 
2. Open the bottom sheet. 
3. Ensure that the `STORY_POST_BOTTOM_SHEET_OPENED` is triggered. 
4. Click Publish. 
5. Ensure that the  `STORY_POST_PUBLISH_TAPPED ` is triggered.
------------
1. Create a new story post. 
2. Open the bottom sheet. 
3. Ensure that the `STORY_POST_BOTTOM_SHEET_OPENED` is triggered. 
4. Dismiss the bottom sheet. 
5. Ensure that the  `PREPUBLISHING_BOTTOM_SHEET_DISMISSED ` is triggered.

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
